_config.yml

```diff
@@ -6,11 +6,4 @@ permalink: none
 destination: site
 exclude: ['README.md','content']
 keep_files: ['docs']
-
-# The recommended way of viewing the website on your local machine is via jekyll using
```

Comment:
This info is out of date. `jekyll serve` works as intended without modifying this, and the site URL should be the full host / path